src/reversion_rest_framework/mixins.py

```diff
@@ -15,11 +16,24 @@ class BaseHistoryModelMixin:
 class HistoryOnlyMixin(BaseHistoryModelMixin):
     @action(detail=True, methods=['GET'], name='Get History')
     def history(self, request, pk=None):
-        object = self.get_object()
-        versions = Version.objects.get_for_object(object)
+        instance = self.get_object()
+        versions = Version.objects.get_for_object(instance).order_by(
```

Comment:
not sure about the ordering... the other way round might make more sense?

Reply:
Well, the order might be in `-revision__date_created`, fixed!